lib/modules/dosomething/dosomething_metatag/dosomething_metatag.module

```diff
@@ -121,8 +121,11 @@ function dosomething_metatag_get_permalink_vars() {
   $rbid = arg(1);
   $fid = ($_GET['fid']);
   $rb_entity = reportback_load($rbid);
-  if (!$fid) {
-    $fid = $rb_entity->fids[0];
+  if (!$fid && !empty($rb_entity->fids)) {
+    $fid = array_pop($rb_entity->fids);
+  }
+  else {
+    $fid = array_pop($rb_entity->getFids());
```

Comment:
Right?! I've been banging my head against the wall on this one today. I haven't been able to figure out why it happens only sometimes. It's _very_ odd.